plugins/modules/monitoring/monit.py

```diff
@@ -140,7 +140,7 @@ def exit_success(self, state):
 
     @property
     def command_args(self):
-        return "-B" if self.monit_version() > (5, 18) else ""
+        return "-B" if self.monit_version() > (5, 18) else None
```

Comment:
Why not
```suggestion
        return ["-B"] if self.monit_version() > (5, 18) else []
```
and use list concatenation instead of post-processing the list when this is used?

Reply:
Makes sense, on it.